frontend/public/components/dashboard/status-card/alert-item.tsx

```diff
@@ -48,7 +48,7 @@ export const StatusItem: React.FC<StatusItemProps> = ({
 
 export const AlertItem: React.FC<AlertItemProps> = ({ alert }) => {
   const LinkComponent = React.useCallback(
-    () => <Link to={alertURL(alert, alert.rule.id)}>View more</Link>,
+    () => <Link to={alertURL(alert, alert.rule.id)}>View Details</Link>,
```

Comment:
```suggestion
    () => <Link to={alertURL(alert, alert.rule.id)}>View details</Link>,
```

Reply:
Thank you for the reviews @andybraren I have updated the PR. :)